### ISSUE TITLE
Ability to remove the `all` section

### ISSUE THREAD

**Issue by lucasmerat** [chrislopresto/ember-freestyle#367]
Hi there. We're having some issues with our freestyle where going to the `all` subsection of components is causing errors. In addition, it's super slow to load every component we have. At this point, `/freestyle` takes the user to `all` by default. Is there a way to configure things so that it would take the user to our first subsection of components instead?

**Comment by lukemelia** [comment]
Hi @lucasmerat,

In order to default to a particular section, you could override the freestyle service like so:

```js
// app/services/ember-freestyle.js
import BaseService from 'ember-freestyle/services/ember-freestyle';

export default BaseService.extend({
  section: 'Your Default Section Name'
});
```

Suppressing `All` in the menu would require an enhancement to the addon. I imagine a boolean property on the service `menuIncludesAllItem` that defaults to `true`. We would check this property in the freetyle-menu component to determine whether to render the All menu item. We would be happy to review a PR achieving this if you would like to work on it.

**Comment by lucasmerat** [comment]
Great! Thanks for the tip. Ill look into opening a PR in the next week or 2.

**Comment by lukemelia** [comment]
Note: work in progress in #372

**Comment by bertdeblock** [comment]
Closing since #372 was merged and released in v0.12.8.
